pom.xml

```diff
@@ -24,7 +24,7 @@
         <spring-security.version>5.2.2.RELEASE</spring-security.version> <!-- sync with version used by spring-boot-->
         <hibernate.version>5.4.10.Final</hibernate.version>
         <hibernate-validator.version>6.0.18.Final</hibernate-validator.version>
-        <postgresql.driver.version>42.2.9</postgresql.driver.version>
+        <postgresql.driver.version>42.2.23</postgresql.driver.version>
```

Comment:
can/should we move to 42.2.24 that has been released in the meantime?

Reply:
https://jdbc.postgresql.org/documentation/changelog.html#version_42.2.24

Reply:
done!